lua/neogit/buffers/log_view/init.lua

```diff
@@ -140,9 +141,24 @@ function M:open()
           local dv = require("neogit.integrations.diffview")
           dv.open("log", stack[#stack].options.oid)
         end,
+        ["b"] = require("neogit.popups").open("branch", function(p)
```

Comment:
Oh, shoot, does this work?? Very cool. I guess it would be good to require popups as a local, and use the same open function pattern for the cherry pick and revert popups.

Oh, and add branch to visual mode too (you were probably already doing that though :p)